packages/c-skip-nav/src/c-skip-nav.ts

```diff
@@ -0,0 +1,110 @@
+/**
+ * Hey! Welcome to @chakra-ui/vue-next CSkipNav
+ *
+ * Component tandem to allow users to skip navigation content
+ *
+ * @see Docs     https://next.vue.chakra-ui.com/c-skip-nav
+ * @see Source   https://github.com/chakra-ui/chakra-ui-vue-next/blob/master/packages/c-skip-nav/src/c-skip-nav/c-skip-nav.ts
+ * @see WebAIM   https://webaim.org/techniques/skipnav/
+ */
+
+import { h, defineComponent } from "vue"
+import {
+  chakra,
+  ComponentWithProps,
+  DeepPartial,
+  HTMLChakraProps,
+  SystemStyleObject,
+  useStyleConfig,
+} from "@chakra-ui/vue-system"
+import { CBox } from "@chakra-ui/vue-next"
+
+const FALLBACK_ID = "chakra-skip-nav"
+
+export interface CSkipNavLinkProps extends HTMLChakraProps<"a"> {}
+
+export const CSkipNavLink: ComponentWithProps<DeepPartial<CSkipNavLinkProps>> =
+  defineComponent({
+    name: "CSkipNavLink",
+    props: {
+      id: {
+        type: String,
+        default: FALLBACK_ID,
+      },
+    },
+    setup(props, { slots, attrs }) {
+      function getBaseStyles(styles: any): SystemStyleObject {
+        return {
+          userSelect: "none",
+          border: "0",
+          borderRadius: "md",
+          fontWeight: "semibold",
+          height: "1px",
+          width: "1px",
+          margin: "-1px",
+          padding: "0",
+          outline: "0",
+          overflow: "hidden",
+          position: "absolute",
+          clip: "rect(0 0 0 0)",
+          ...styles,
+          _focus: {
+            clip: "auto",
+            width: "auto",
+            height: "auto",
+            boxShadow: "outline",
+            padding: "1rem",
+            position: "fixed",
+            top: "1.5rem",
+            insetStart: "1.5rem",
+            ...styles["_focusVisible"],
+          },
+        }
+      }
```

Comment:
This might be better as a computed, otherwise the changes in styles won't affect this and it's also a little bit more optimized 😄 

Something like this would do the trick : 

```ts
const baseStyles = computed(() => ({
 ...yourStyles,
 ...styles.value,
 ...otherStyles
})
```